semantic-conventions/src/tests/data/markdown/deprecated/expected.md

```diff
@@ -16,7 +16,7 @@
 
 **[1]:** If `net.transport` is not specified, it can be assumed to be `IP.TCP` except if `http.flavor` is `QUIC`, in which case `IP.UDP` is assumed.
 
-`http.flavor` MUST be one of the following or, if none of the listed values apply, a custom value:
+`http.flavor` If one of the listed values applies, then the respective value MUST be used, otherwise a custom value MAY be used:
```

Comment:
This reads quite strange. Before the change it was a "natural" sentence but now it's just appended to the name of the attribute.

How about

> The following values are defined for `http.flavor`. If one of them applies, then ...

or at least adding a colon

> `http.flavor`: If one of the values listed below applies, then the respective value MUST be used, otherwise a custom value MAY be used:

Reply:
makes sense, ptal